include/swift/Basic/InputFile.h

```diff
@@ -0,0 +1,66 @@
+//===--- InputFile.h --------------------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+
+#ifndef InputFile_h
```

Comment:
Please use the same header guard style as the other headers: SWIFT_BASIC_INPUTFILE. (Also, why is this in Basic instead of Frontend?)

Reply:
Fixed in all my new files. Moved from Basic to Frontend. (Had thought other parts of the compile would need that class, but at least they don't yet.)